src/platform/site-wide/sass/shame.scss

```diff
@@ -532,19 +534,20 @@ img[data-srcset] {
   position: relative;
 }
 
-.usa-grid > :last-child, .usa-grid-full > :last-child {
+.usa-grid > :last-child,
+.usa-grid-full > :last-child {
   &.half-em-bottom-margin {
-    margin-bottom: .5em;
+    margin-bottom: 0.5em;
   }
 }
 
 .inline-table-helper {
-  display: inline-table
+  display: inline-table;
```

Comment:
haha if this were missing, then that means the styles weren't being applied likely, i wonder how this will change things! 😬

Reply:
gotta love finding missing bits like that

Reply:
Not certain but I _think_ the SCSS compiler would've grabbed it or accepted it since there isn't a rule on the next line. I'm not sure `inline-table` looks like though anyway 🤷